src/app/geolocation.service.ts

```diff
@@ -1,10 +1,15 @@
 import { EventEmitter, Injectable, OnInit } from '@angular/core';
-import { BehaviorSubject } from 'rxjs';
+import { toJSDate } from '@ng-bootstrap/ng-bootstrap/datepicker/ngb-calendar';
+import { BehaviorSubject, lastValueFrom, take } from 'rxjs';
+import { WeatherService } from './services/weather.service';
 
 @Injectable({
   providedIn: 'root'
 })
 export class GeolocationService {
+
```

Comment:
no space (this file may go bye bye anyway tho!)

Reply:
![image](https://user-images.githubusercontent.com/106711744/208813426-e2ddabd4-9387-4334-a8fa-e7d5ba1d786c.png)